packages/components/organisms/f-checkout/test/specs/accessibility/axe-accessibility.spec.js

```diff
@@ -1,15 +1,43 @@
 import { getAccessibilityTestResults } from '../../../../../../../test/utils/axe-helper';
-import CheckoutComponent from '../../../test-utils/component-objects/f-checkout.component';
+const Checkout = require('../../../test-utils/component-objects/f-checkout.component');
+const checkout = new Checkout();
+
 
 describe('Accessibility tests', () => {
-    beforeEach(() => {
-        browser.url('?path=/story/components-organisms--checkout-component');
-        browser.switchToFrame(0);
-        CheckoutComponent.waitForCheckoutComponent();
+
+    it('a11y - should test f-checkout component (delivery) WCAG compliance', () => {
+        // Act
+        const checkoutData = {
+            type: 'delivery', 
+            isAuthenticated: true, 
+            isValid: true
+        }
+        checkout.open(checkoutData);
+        checkout.waitForComponent();
+        const axeResults = getAccessibilityTestResults('f-checkout-delivery');
```

Comment:
Does this method also assert on the results? Maybe the naming could be updated slightly

Reply:
I see this is already quite widely used, and it's very minor, so I don't think it's an issue.